bluepyemodel/model/neuron_model_configuration.py

```diff
@@ -211,6 +211,7 @@ def init_from_dict(self, configuration_dict, morphology, auto_mechanism=False):
                     mechanism.get("version", None),
                     mechanism.get("temperature", None),
                     mechanism.get("ljp_corrected", None),
+                    mechanism.get("id", None),
```

Comment:
shouldn't we update the definition of `add_mechanism`?

Reply:
good catch. solved in latest commit

Reply:
Thanks, then we should either call the function using keyword arguments or consider rearranging the function definition of `add_mechanism` to move `auto_parameter` to the last position in its parameter list

Reply:
Indeed. I added keyword arguments to the function call